dotcom-rendering/src/web/browser/getName.ts

```diff
@@ -3,11 +3,11 @@
  *
  * We expect the element to always be a `gu-*` custom element
  *
- * @param marker : The html element that we want to read the name attribute from;
+ * @param element : The html element that we want to read the name attribute from;
```

Comment:
Respecting a comment made by @sndrs [over in libs](https://github.com/guardian/libs/pull/297#discussion_r755031903) for consistency